server/sonar-server/src/main/java/org/sonar/server/qualitygate/ws/CreateConditionAction.java

```diff
@@ -72,18 +77,15 @@ public void define(WebService.NewController controller) {
 
   @Override
   public void handle(Request request, Response response) {
-    userSession.checkPermission(QUALITY_GATE_ADMIN);
+    userSession.checkLoggedIn().checkOrganizationPermission(defaultOrganizationProvider.get().getUuid(), QUALITY_GATE_ADMIN);
```

Comment:
same comment as earlier, addition of `checkLoggedIn()` introduces a change in behavior which is not mandatory